docs/index.html

```diff
@@ -27,10 +27,33 @@ <h1 style="border-bottom: 1px solid var(--main-header-border-bottom-color, #eaec
           Quick Start
         </div>
         <div class="description">
-          <p>Pick an IDE to get started on any C++ project.</p>
+          <p>First, set up your command line for your machine. Then, pick an IDE to get started on any C++ project. In 280, we strongly recommend <a href="setup_vscode.html">VS Code</a>.</p>
         </div>
         <div class="ui hidden divider"></div>
-        <div class="ui three stackable cards">
+        <div class="ui two stackable cards">
+          <a class="ui raised card" target="_blank" href="setup_vscode.html">
+            <div class="content">
+              <div class="header">
+                <i class="icon windows"></i></i> Windows Setup
+              </div>
+              <div class="description">
+                <p>Basic setup on Windows</p>
+              </div>
+            </div>
+          </a>
+          <a class="ui raised card" target="_blank" href="setup_xcode.html">
+            <div class="content">
+              <div class="header">
+                <i class="icon apple"></i> Mac Setup
```

Comment:
```suggestion
                <i class="icon apple"></i> macOS Setup
```
For consistency with title below.

Reply:
fixed, thanks!